conf/application.conf

```diff
@@ -153,6 +153,8 @@ vat-agent-client-lookup-frontend {
   host = "http://localhost:9147"
   startUrl = "/vat-through-software/submit-vat-return/test-only/vaclf-stub"
   unauthorisedUrl = "/vat-through-software/submit-vat-return/test-only/vaclf-stub-unauth"
+  changeClientUrl = "/vat-through-software/representative/client-vat-number"
```

Comment:
Should this go to a stub page locally? If so can you add a app-config pr to overwrite this for QA and Production. Same below for the `whatToDoUrl`